### PR TITLE
kraft: Change unikraft and lwip version to stable in kraft.yaml

### PR DESCRIPTION
**Pull request by eduardvintila**
Signed-off-by: Eduard Vintilă <eduard.vintila47@gmail.com>

### CODE REVIEW

**Review by StefanJum — APPROVED** [review]
Works fine, thank you!
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>

File: kraft.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 specification: '0.5'
```
Comment (razvand) [inline kraft.yaml:2]:
```suggestion
---
specification: '0.5'
```

Compatible with the new syntax

File: kraft.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 name: httpreply
 unikraft:
-  version: '0.5'
+  version: '0.10.0'
```
Comment (razvand) [inline kraft.yaml]:
```suggestion
  version: stable
```

File: kraft.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ targets:
     platform: kvm
 libraries:
   lwip:
-    version: '0.5'
+    version: '0.10.0'
```
Comment (razvand) [inline kraft.yaml]:
```suggestion
    version: stable
```

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @eduardvintila . See my remarks.

**Review by razvand — APPROVED** [review]
This works. Thanks, @eduardvintila 

Reviewed-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>
Approved-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>

**Pull request closed without merging**